privaterelay/templates/faq.html

```diff
@@ -19,7 +19,7 @@ <h2 class="faq-headline">What about spam?</h2>
             While Relay does not filter for spam, our email partner Amazon SES does block spam and malware. If Relay forwards messages you don’t want, you can update your Relay settings to block messages from the alias forwarding them.
           </p>
           <p class="faq-answer">
-            If you see a broader problem of unwanted email from all of your aliases, please report this to us so we can consider adjusting the SES spam thresholds for this service.
+            If you see a broader problem of unwanted email from all of your aliases, please <a class="text-link" rel="noopener noreferrer" href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank">report this to us</a> so we can consider adjusting the SES spam thresholds for this service.
```

Comment:
Can we remove `noreferrer` from this link? addons.mozilla.org is one of our sites already, and the referrer can help us learn how much of the traffic to our AMO listing is coming from our site links.
```suggestion
            If you see a broader problem of unwanted email from all of your aliases, please <a class="text-link" rel="noopener" href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank">report this to us</a> so we can consider adjusting the SES spam thresholds for this service.
```